Makefile

```diff
@@ -28,10 +28,10 @@ start-test-dbs:
 
 .PHONY: restart-db
 restart-db:
```

Comment:
Should we also restart the MongoDB?

If so, then I think we should change the command name to be `restart-test-dbs` so it's clear it's both databases and it's for the test ones.

If not, then I think we should change the command name to be `restart-test-postgres-db` or `restart-test-pg-db` to be clear it's just for the PostgreSQL database and it's for the test ones.

Reply:
Not sure if we need to restart the mongo db 🤔.

I haven't had any issues related to out of sync mongo DBs.

We could rename this to be clearer that this would affect only the Postgres DB and maybe if we have issues with inconsistent mongo DBs, we add a command for it later?

Reply:
Coooool, let's go with what you suggested! 🚢